### PR TITLE
[guide] add guide for DirSheet

### PR DESCRIPTION
**Pull request by anjakefala**
#2313

Screenshot from 2024-02-25 22-48-35

### CODE REVIEW

File: visidata/guides/DirSheet.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+sheettype: DirSheet
+---
+# Directory Sheet
+
+The **DirSheet** is a display of files and folders in the *source* directory.
+
+To load a **DirSheet**:
+
+- provide VisiData a directory as input (e.g. [:code]vd sample_data/[/])
```
Comment (saulpw) [inline visidata/guides/DirSheet.md]: - don't have to mention "VisiData" in guides.  "provide a directory as a source on the CLI"

File: visidata/guides/DirSheet.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- {help.commands.open_dir_current} (equivalent to [:code]vd .[/])
+
+Pass the flag [:code]-r[/] ([:code]--recursive[/]) to display all of the files in all subfolders.
```
Comment (saulpw) [inline visidata/guides/DirSheet.md]: We should probably mention options.dir_depth here

File: visidata/guides/DirSheet.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+Basic usage of the **DirSheet** is to find and load files within VisiData:
```
Comment (saulpw) [inline visidata/guides/DirSheet.md]: avoid passive voice.  "Use the **DirSheet** to find and open files."

File: visidata/guides/DirSheet.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+- {help.commands.open_row_file}
+- {help.commands.open_rows}
+- ([:longname]open-dir-parent[/]) to open parent directory
```
Comment (saulpw) [inline visidata/guides/DirSheet.md]: can we use `{help.commands.open_dir_parent}` here?

File: visidata/guides/DirSheet.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+- {help.commands.sysopen_row}
+
+Advanced usage of **DirSheet** is as a file systems manager.
```
Comment (saulpw) [inline visidata/guides/DirSheet.md]: "Use the **DirSheet** as a file manager:"

File: visidata/guides/DirSheet.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+- {help.commands.copy_row}
+- {help.commands.copy_selected}
+- [:keys]d[/] ([:longname]delete-row[/]) to delete file from filesystem
+- [:keys]e[/] ([:longname]edit-cell[/]) to change file metadata (all of the file metadata, except for *filetype*, is modifiable)
+
+Modifications on the **DirSheet** are deferred - they do not take effect on the filesystem itself until they are confirmed.
+- [:keys]z Ctrl+S[/] ([:longname]commit-sheet[/]) to commit all changes to the filesystem (not undoable!)
```
Comment (saulpw) [inline visidata/guides/DirSheet.md]: why not "{help.commands.commit_sheet}"?

Comment (saulpw) [reply visidata/guides/DirSheet.md]: ah I think maybe we should experiment with making commit-sheet an identical command but with a different helpstr on this sheet

File: visidata/guides/DirSheet.md (hunk 4 above)
Comment (anjakefala) [reply visidata/guides/DirSheet.md]: Yes! I think this was before we added support for unbound.

Comment (anjakefala) [reply visidata/guides/DirSheet.md]: Oh no! It was because the keystroke `\`` broke things in the past. It seems to render correctly now.

**Review by saulpw — APPROVED** [review]
This looks good, thank you @anjakefala !

**Pull request merged**